scout/reports.md

```diff
@@ -0,0 +1,100 @@
+---
+description: More details on the Docker Scout Advisory Database and CVE-to-package matching service.
+keywords: scanning, analysis, vulnerabilities, Hub, supply chain, security, report, reports
+title: Analysis reports
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout reports help you share a report of the analysis of images in a namespace with your team. The reports helps team members in roles such as security, compliance, and operations to know what vulnerabilities and issues they need to focus on.
+
+## Overview
+
+![A screenshot of the Docker Scout vulnerabilities overview](./images/reports-overview.png)
+The overview tab shows the number of critical, high, medium, and low vulnerabilities over time.
+
+## Images
+
+![A screenshot of the Docker Scout images list for a namespace](./images/reports-images.png)
+
+The images tab shows a list of images in a namespace. You can sort the list by source repository name and search for specific repositories using the search box.
+
+Each entry in the list shows the following details:
+
+- The repository for the image. Clicking the link for the repository opens the [list of tags for the repository](#).
+- The most recent version of the image and the vulnerabilities for that version. Clicking the link for the version opens a list of all versions for the repository.
+- The operating system and architecture of the image.
+- The base image and version used by the repository and the vulnerabilities for that version. Clicking the link for the base image opens [the image layer view](#image-layer-view).
+- The recommended fixes, which can include options such as changing tags or rebuilding an image.
+- The predicted improvement to the vulnerabilities if you apply the recommended fixes.
+- An action button to show full details of [recommended fixes](#recommended-fixes).
```

Comment:
**suggestion:** full details might be a bit misleading, I understand you mean it's full when compared with the inline info we show on the table itself. But when compared with the image layer view it's the same, so maybe we could reword this somehow to make that distinction clear?